docs/CQ.rst

```diff
@@ -0,0 +1,59 @@
+Create a Contribution Questionnaire
+===================================
+
+All the new dependencies and all the dependencies updates should be submited as a new CQ into the
+`ECLIPSE IP Zilla <https://dev.eclipse.org/ipzilla/query.cgi>`__.
+
+Submitting a CQ
+---------------
+
+Visit `https://projects.eclipse.org/projects/locationtech.geotrellis/ <https://projects.eclipse.org/projects/locationtech.geotrellis/>`__
+and log into the system.
+
+.. figure:: img/locationtech-geotrellis.png
+   :alt: The GeoTrellis project page
+
+On the right, in the ``COMMITTER TOOLS`` side bar, you can find a link `Create a Contribution Questionnaire <https://projects.eclipse.org/projects/locationtech.geotrellis/cq/create>`__
+
+.. figure:: img/geotrellis-cq.png
+   :alt: Create a Contribution Questionnaire page
+
+Next, we need to create a Third-Party Code Request usually. All the next steps would be based on a particular library example.
```

Comment:
I think this would sound better as: "Next, we need to create a Third-Party Code Quest. For this example, we'll be creating a CR for `pureconfig v0.10.2`".